core/dev/config/console-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.13.0-rc.1",
+  "minimumCoreVersion": "4.0.0",
+  "protocol": "https",
+  "host": "{enter-openhim-core-service-url-here}",
```

Comment:
@bradsawadye Are you going to make use of Ingress? If so then this can be updated to the Ingress URLs